resources/assets/js/components.tsx

```diff
@@ -8,6 +8,6 @@
 // require('./components/ReferenceList');
 require("./components/ApplicationReview/ReviewApplicationsContainer");
 require("./components/ApplicationReview/ApplicationReviewContainer");
-require("./components/ApplicantSkills/WordCounter");
+require('./components/WordCounter/WordCounter');
```

Comment:
First, I believe WordCounter should be replaced with WordCounterWrapper now.

But more importantly, we've decided recently to try to break our js into seperate bundles, to reduce the amount of unnecessary code downloaded by the browser for a given page. In this case, it means that _instead_ of adding WordCounter to `components.tsx`, you should export WordCounterWrapper in `webpack.mix.js`, and include this specific script in the twig template. Check `job-builder-intro.html.twig` for an example of how this is done.